data/mods/Magiclysm/Spells/black_dragon.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "type": "SPELL",
+    "id": "acid_claw",
+    "name": "Acid Claw",
+    "description": "Your claws drip with acid as you make a slash attack.",
+    "valid_targets": [ "hostile" ],
+    "flags": [ "SOMATIC", "VERBAL", "NO_LEGS" ],
+    "spell_class": "DRAGON_TALONS_BLACK",
+    "effect": "target_attack",
+    "damage_type": "acid",
+    "max_level": 4,
+    "min_damage": 10,
+    "max_damage": 50,
+    "damage_increment": 10.0,
+    "min_range": 1,
+    "max_range": 1,
+    "difficulty": 0,
+    "base_casting_time": 100,
+    "energy_source": "STAMINA",
+    "base_energy_cost": 100,
+    "extra_effects": [ { "id": "dragon_claw" } ]
+  },
+  {
+    "type": "SPELL",
+    "id": "dragon_claw",
+    "name": "Dragon's Claw",
+    "description": "The claw damage in Acid Claw.",
+    "valid_targets": [ "hostile" ],
+    "effect": "target_attack",
+    "damage_type": "cut",
+    "min_damage": 20,
+    "max_damage": 20
+  },
+  {
+    "type": "SPELL",
+    "id": "acid_bite",
+    "name": "Acid Bite",
```

Comment:
this is a carbon-copy of the other spell. what this does is just clog up the spell list; it needs some kind of uniqueness that sets it apart from the other one. possibly even not make it a spell and just make it a bite attack that deals acid damage.

Reply:
The last time I attempted to put acid damage on the bite attack in the mutation, it didn't do anything. I was under the impression that it was still not possible, but perhaps though an enchantment it may be. I'll take a look.

Reply:
if it doesn't i will help you get that working properly.

Reply:
I made some adjustments and I took out the old claw spell and am in the process of making it an acid cone associated with the claws. I'll need to work on a field and stuff too, but the plan is to change Acid Claw to a cone spell and leave Acid Bite as it is.